Site/SiteMultipartMailMessage.php

```diff
@@ -337,6 +340,46 @@ public function addBcc($email)
 	}
 
 	// }}}
+	// {{{ public function getCcList()
+
+	/**
+	 * Gets an array of email addresses for CC
+	 *
+	 * @return arrays $email addresses for CC.
+	 */
+	public function getCcList()
+	{
+		$list = array();
+		foreach ($this->cc_list as $email) {
+			if (SwatString::validateEmailAddress(trim($email))) {
```

Comment:
This may do too much validation. The comment on that method says that it will not validate emails that don't have a TLD. Not sure if we want to be that restrictive.